sdk/communication/Azure.Communication.Common/src/CommunicationTokenRefreshOptions.cs

```diff
@@ -39,6 +44,24 @@ public class CommunicationTokenRefreshOptions
             Argument.AssertNotNull(tokenRefresher, nameof(tokenRefresher));
             RefreshProactively = refreshProactively;
             TokenRefresher = tokenRefresher;
+            RefreshOffsetTime = new TimeSpan( 0, 0, DefaultExpiringOffsetMinutes, 0);
+        }
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="CommunicationTokenRefreshOptions"/>.
+        /// </summary>
+        /// <param name="refreshProactively">Indicates whether the token should be proactively renewed prior to expiry or renew on demand.</param>
+        /// <param name="tokenRefresher">The function that provides the token acquired from CommunicationIdentityClient.</param>
+        /// <param name="refreshOffsetTime">Proactive refresh interval.</param>
+        public CommunicationTokenRefreshOptions(
```

Comment:
you can probably get rid of the new constructor by just making the refreshOffsetTime the last argument and making it [optional](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments)

if not, you can definitely refactor the constructors in a way that once calls the other using `:this()`